receiver/splunkhecreceiver/receiver.go

```diff
@@ -45,12 +45,16 @@ const (
 	responseOK                        = "OK"
 	responseInvalidMethod             = `Only "POST" method is supported`
 	responseInvalidEncoding           = `"Content-Encoding" must be "gzip" or empty`
+	responseInvalidDataFormat         = `{"text":"Invalid data format","code":6}`
+	responseErrEventRequired          = `{"text":"Event field is required","code":12}`
+	responseErrEventBlank             = `{"text":"Event field cannot be blank","code":13}`
 	responseErrGzipReader             = "Error on gzip body"
 	responseErrUnmarshalBody          = "Failed to unmarshal message body"
 	responseErrInternalServerError    = "Internal Server Error"
 	responseErrUnsupportedMetricEvent = "Unsupported metric event"
 	responseErrUnsupportedLogEvent    = "Unsupported log event"
```

Comment:
Do we want to revise the existing error message to follow similar structure too? For example `Internal Server Error` would be error code 8. If the answer is yes, a further question to @atoulme, this would change the behavior for existing users, do you know who we need to notify to get signed off for?

Reply:
I think you can make that change. The config options for beta components are considered stable, but we have no particular consideration for error messages, so we should be ok changing them as long as there is a release note for it.